newswires/client/src/App.tsx

```diff
@@ -91,7 +92,20 @@ export function App() {
 											padding-bottom: 3px;
 										`}
 									>
-										<h1>Newswires</h1>
+										<h1>
+											<EuiLink
+												href="/feed"
+												external={false}
+												css={css`
+													color: inherit;
+													&:hover {
+														text-decoration: none;
```

Comment:
looks good to me, though I'd suggest we should have the text underlined on hover for accessibility purposes

currently the 'Newswires' text is bolder on `main`, but if it's an intentional choice to make it less bold here then I'm happy to go with that 👍